language-support/java/codegen/src/it/daml/Tests/NumericTest.daml

```diff
@@ -1,9 +1,10 @@
 -- Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 -- SPDX-License-Identifier: Apache-2.0
-
+{-# LANGUAGE CPP #-}
```

Comment:
Can we just exclude the whole file from the sources for LF < 1.7 instead of throwing CPP at this?